### ISSUE TITLE
Intel Iris 6100 - 10.12.5 - 3840x2160@60Hz ??

### ISSUE THREAD

**Issue by d0cm0d** [Floris497/mac-pixel-clock-patch-V2#192]
Hello, 

first of all, thank you for your great work.
You made it possible to drive my UHD Samsung TV with 60Hz via Displayport-to-HDMI adapter.
At the moment, I am running Mac OS 10.12.3 and I am a little bit in fear that upgrading to 10.12.5 will kill the 60Hz.

Here is a little information about my system graphics:

Intel Iris Graphics 6100:
  Chipsatz-Modell:	Intel Iris Graphics 6100
  Typ:	GPU
  Bus:	Integriert
  VRAM (dynamisch, maximal):	1536 MB
  Hersteller:	Intel (0x8086)
  Geräte-ID:	0x162b
  Versions-ID:	0x0009
  Metal:	Unterstützt

Displays:

Farb-LCD:
  Displaytyp:	Retina LCD
  Auflösung:	2560 x 1600 Retina
  Retina:	Ja
  Pixeltiefe:	32-Bit Farbe (ARGB8888)
  Synchronisierung:	Aus
  Eingeschaltet:	Ja
  Helligkeit automatisch anpassen:	Ja
  Integriert:	Ja

SAMSUNG:
  Auflösung:	3840 x 2160 @ 60 Hz
  Pixeltiefe:	32-Bit Farbe (ARGB8888)
  Hauptdisplay:	Ja
  Synchronisierung:	Aus
  Eingeschaltet:	Ja
  Rotation:	Unterstützt
  Helligkeit automatisch anpassen:	Nein
  Verbindungstyp:	DVI oder HDMI
  Adapter Firmware-Version:	7,26
  Fernseher:	Ja

otool -t CoreDisplay |tail -n +2 |md5 -q
e9d7a42b6613a45a69a41e8099d0e369

Can I update without fear or should I wait?
Thanks for your help,
best regards,
Pascal

**Comment by Floris497** [comment]
I would not be able to promise anything :/ 10.12.5 seems to have problems but mostly with dedicated Nvidia GPU’s

**Comment by Floris497** [comment]
How is it looking now? is it working? New patch for High Sierra is working but still testing it.. will release soon

**Comment by d0cm0d** [comment]
I didn't test it. Still running 10.12.3 because I don't want to loose 60Hz support.
Are there any news concerning the patch for High Sierra?
Thanks, best regards, Pascal

**Comment by Floris497** [comment]
Yep, CoreDisplay for 10.13 is here for some time, should be working.

**Comment by d0cm0d** [comment]
Thank you very much!!
I just updated to 10.13.2 and I get 3840 x 2160 @ 60 Hz, so I can confirm that the patch is working.

Best regards, Pascal
